### PR TITLE
Social Media Buttons: Add Phone and resolve #725

### LINKED ISSUE #725: Social Media Buttons: Change title text for email button
The Email network option doesn't really make much sense with the default title text - `SITENAME on Envelope`. We should change it to something more logical like, `Email SITENAME`.

[Related](https://siteorigin.com/thread/social-media-buttons-3/)

### PR DESCRIPTION
**Pull request by AlexGStapleton**
This PR adds a new Phone button and resolves #725 while also accounting for the addition of the Phone "network".

### CODE REVIEW

File: widgets/social-media-buttons/tpl/default.php [hunk 1]
```diff
@@ -6,7 +6,19 @@
 		if( !empty($instance['design']['hover']) ) $classes[] = 'ow-button-hover';
 		$classes[] = "sow-social-media-button-" . sanitize_html_class( $network['name'] );
 		$classes[] = "sow-social-media-button";
-		$title = empty( $network['icon_title'] ) ? sprintf( __( '%s on %s', 'so-widgets-bundle' ), get_bloginfo( 'name' ), ucwords( str_replace( '-', ' ', $network['name'] ) ) ) : $network['icon_title'];
+
+		if( ! empty( $network['icon_title'] ) ) {
+			$title = $network['icon_title'];
+		} else {
+			if ( $network['name'] == 'email' || $network['name'] == 'phone' ) {
+				if ( $network['title'] == 'envelope' ) $network['name'] == 'email';
```
Comment (braamgenis) [inline widgets/social-media-buttons/tpl/default.php]: This is quite confusing. Where does the `$network['title']` come from? And then it's just doing a check to see if `$network['name']` is equal to 'email' without assigning it to anything?

File: widgets/social-media-buttons/tpl/default.php [hunk 2]
```diff
@@ -6,7 +6,19 @@
...
+
+				/* translators: Social Media Button with a verb and then site name. As an example, Call Eastlink Windows */
+				$title = sprintf( __( '%s %s', 'so-widgets-bundle' ), ucfirst( $network['name'] ), get_bloginfo( 'name' ) );
```
Comment (braamgenis) [inline widgets/social-media-buttons/tpl/default.php]: I think the below format could work for email and phone as is, e.g. Eastlink Windows on Email, but regardless, perhaps we should add a field to let users specify the format they want. Then we can just have our one as a default.

Comment (AlexGStapleton) [reply widgets/social-media-buttons/tpl/default.php]: We do actually already have a way for users to specify a custom title, icon_title. The issue with that though is we can't expect users to always fill that field in just to correct the title - for example, "Envelope on Eastlink Windows" is obviously wrong.

With that said, I do like the Sitename on type name a little more than "Email Eastlink Windows", but the problem with that is that both buttons use tel and mailto. By clicking that link you're not taken to somewhere, you're doing something - be it calling or emailing.

File: widgets/social-media-buttons/tpl/default.php (hunk 1 above)
Comment (AlexGStapleton) [reply widgets/social-media-buttons/tpl/default.php]: That actually contains two major typos. 😞 
It's not email, it's Envelope. It's also not `$network['title']`, it's `$network['name']`. I had thought I fixed that before committing it but I guess I didn't - sorry about that!

**Pull request merged**